test/gradcheck.jl

```diff
@@ -93,6 +93,15 @@ end
 @test gradtest((x, W, b) -> relu.(W*x .+ b), (5,3), (2,5), 2)
 @test gradtest((x, W, b) -> selu.(W*x .+ b), 5, (2,5), 2)
 @test gradtest((x, W, b) -> selu.(W*x .+ b), (5,3), (2,5), 2)
+@test gradtest((x, W, b) -> elu.(W*x .+ b, 2), 5, (2,5), 2)
+@test gradtest((x, W, b) -> elu.(W*x .+ b, 2), (5,3), (2,5), 2)
+
+# tests for https://github.com/FluxML/Zygote.jl/issues/758
+@test gradient(xs -> sum(selu.(xs)), [1_000, 10_000]) == ([1.0507009873554805, 1.0507009873554805],)
```

Comment:
Can we use gradtest to check against numerical differentiation

Reply:
I have gradtest for selu on rows 94-95, and I was kinda lost in the gradtest functions, so I didn't understand how can I pass some specific numbers to gradtest, not just matrix sizes

Reply:
See `gradcheck`

Reply:
Oh, I see, thanks.
Ok, now I'm a bit lost :D 
I added some tests, but they are failing because the numeric gradient is inexact:
```julia
ulia> gradcheck(x->sum(selu.(x)),[1_000, 10_000])
ERROR: InexactError: Int64(999.9999999925494)
Stacktrace:
 [1] Int64 at .\float.jl:710 [inlined]    
 [2] convert at .\number.jl:7 [inlined]   
 [3] setindex! at .\array.jl:847 [inlined]
 [4] ngradient(::Function, ::Array{Int64,1}) at C:\Projects\others\Zygote.jl\test\gradcheck.jl:14
 [5] gradcheck(::Function, ::Array{Int64,1}) at C:\Projects\others\Zygote.jl\test\gradcheck.jl:24
 [6] top-level scope at none:1
```

Reply:
And when I make them floats and not integers, the `gradcheck` is returning false
`gradcheck(x->sum(selu.(x)),[1_000., 10_000.])`

Reply:
Or should I assert that the gradcheck is returning false to prove the instability and thus different results of derivation?

Reply:
For 
```julia
gradcheck(x->sum(selu.(x)),[100., 1_000.])
gradcheck(x->sum(elu.(x, 3.5)),[100., 1_000.])
```
it's working, so should I put these smaller numbers here?

Reply:
It should give the right results with floats, I would think

Reply:
It shouldn't fail with large numbers either

Reply:
I would verify that it's the inexact behavior in `ngradient` causing this

Reply:
It's not working for larger numbers, see
```
julia> ngradient(x->sum(selu.(x)),[1_000., 10_000.])
([1.0506591796875, 1.0506591796875],)

julia> gradient(x->sum(selu.(x)),[1_000., 10_000.])
([1.0507009873554805, 1.0507009873554805],)

julia> gradient(x->sum(selu.(x)),[1_000., 10_000.]) .- ngradient(x->sum(selu.(x)),[1_000., 10_000.])
([4.1807667980453544e-5, 4.1807667980453544e-5],)
```

Reply:
The difference is bigger than atol, so it returns false.

Reply:
I see, that makes sense, that shouldn't be an issue in this case hopefully, although I would add a `@test_broken` with the reason in there so we can track this.

Reply:
ok, I'll add `@test` for the lower number and `@test_broken` for higher numbers with the comment on that

Reply:
Tests are now passing with new tests and also broken test.